apps/src/componentLibrary/typography/typography.module.scss

```diff
@@ -153,24 +258,60 @@ h6,
 .heading-xs { @include heading-xs; }
 
 // Paragraphs
+// Use on larger paragraphs
+.body-one { @include body-one; }
 // Use on default paragraphs
 p,
-.body-one { @include body-one; }
-// Use on course block descriptions or anywhere
-// else that needs smaller paragraph text
-p.body-two,
 .body-two { @include body-two; }
+// [TODO: Kelby pls approve] (?) Use on course block descriptions or anywhere
+// else that needs smaller paragraph text
+.body-three { @include body-three; }
```

Comment:
@kelbyhawn with our new typography scale -two will be the default size, `-three` is smaller and `-one` is bigger than the default size. Please approve that I've moved your comment on purpose of `body-three` correctly(used to be `body-two`)

Reply:
Looks good!